src/dev-app/tree/tree-demo-module.ts

```diff
@@ -24,7 +24,7 @@ import {ChecklistTreeDemo} from './checklist-tree-demo/checklist-tree-demo';
 import {ChecklistNestedTreeDemo} from './checklist-tree-demo/checklist-nested-tree-demo';
 import {DynamicTreeDemo} from './dynamic-tree-demo/dynamic-tree-demo';
 import {LoadmoreTreeDemo} from './loadmore-tree-demo/loadmore-tree-demo';
-
+import {ExamplePageModule} from '../example/example-module';
```

Comment:
This should not be imported here. This file is auto-generated to import all of the individual example NgModules, so you're creating a circular dependency.

Reply:
This is a different example-module.  The one you talked about is under material-examples.

Reply:
Ah, right, I mixed those up.